dockers/docker-sonic-restapi/start.sh

```diff
@@ -7,4 +7,14 @@ rm -f /var/run/rsyslogd.pid
 
 supervisorctl start rsyslogd
 
-supervisorctl start restapi
+SERVER_CERT=/etc/sonic/certificates/restapiserver.crt
+SERVER_KEY=/etc/sonic/certificates/restapiserver.key
+CLIENT_CERT=/etc/sonic/certificates/restapiclient.crt
+
+while [ : ]
+do
+    if [[ -f $SERVER_CERT && -f $SERVER_KEY && -f $CLIENT_CERT ]]; then
+        supervisorctl start restapi
+    fi
+    sleep 5s
```

Comment:
suggest to increase the 60 seconds, and should generate a syslog info level to waiting for certs to start restapi server ....